ui/app/src/modules/Preview/Tools/ToolPanel.tsx

```diff
@@ -38,13 +38,14 @@ const useStyles = makeStyles((theme: Theme) => createStyles({
 type ToolPanelProps = PropsWithChildren<{
   title: string | MessageDescriptor;
   BackIcon?: ElementType,
-  onBack?: () => void
+  onBack?: () => void,
+  classes?: any
```

Comment:
Classes, as in other components should be an object with the classes for each piece of the ui. e.g. `{ header, body }`; type it.